torch/_inductor/ir.py

```diff
@@ -4677,7 +4702,7 @@ def require_stride1(cls, x):
         return cls.copy_input(x)
 
     @classmethod
-    def require_stride_order(cls, x, order, allow_padding=False):
+    def require_stride_order(cls, x, order, allow_padding=False, actual_strides=None):
```

Comment:
Can we instead add a new api `require_exact_strides` or just `require_strides` instead of adding this argument hee ?

Reply:
added in https://github.com/pytorch/pytorch/pull/130956/commits/6419105c7136f2235d211c2fec38d6fe1fd9a439